packages/api/blocks-api/src/blocks/vimeo-video.block.ts

```diff
@@ -0,0 +1,30 @@
+import { IsBoolean, IsOptional, IsString } from "class-validator";
+
+import { BlockData, BlockDataInterface, BlockInput, createBlock, inputToData } from "./block";
+import { BlockField } from "./decorators/field";
+
+class VimeoVideoBlockData extends BlockData {
+    @BlockField({ nullable: true })
+    vimeoIdentifier?: string;
+
+    @BlockField({ nullable: true })
+    autoplay?: boolean;
+}
+
+class VimeoVideoBlockInput extends BlockInput {
+    @IsOptional()
+    @IsString()
+    @BlockField({ nullable: true })
+    vimeoIdentifier?: string;
```

Comment:
Should we add a validation here?

Reply:
@jomunker please also add server-side validation for the identifier to mach the Vimeo URL/identifier format. Thanks!